docs/connecting-data/connecting-to-apis-and-external-data/servicenow-api.md

```diff
@@ -1,7 +1,150 @@
----
-description: How to Connect DataDistillr to the ServiceNow API
----
-
 # ServiceNow API
 
-Details coming soon!
\ No newline at end of file
+## Creating a ServiceNow account
+Set up an account with [ServiceNow](https://www.servicenow.com/).
+
+
+### Rate Limits
+To prevent excessive inbound REST API requests, set rules that limit the number of inbound REST API requests processed per hour. 
+You can create rules to limit requests for specific users, users with specific roles, or all users.
+
+## How to Connect DataDistillr to ServiceNow
+To set up a data source connection for ServiceNow, you will need to have:
+
+- A unique name for your data source connection to be used in queries
+- The Instance Name associated ServiceNow Instance
+- The Username associated with your ServiceNow Instance
+- The Password associated with your ServiceNow Instance
+
+### Data Source Form
+To locate the ServiceNow form, follow the steps in [Connecting Your Data to DataDistillr](../../). When you get to the window to choose the data source type, select API as shown below.
+
+<figure markdown>
+  ![Data Source Wizard][image-7]{ width="100%" }
+</figure>
+
+
+
+On the API screen, select ServiceNow from the list of API forms.
+
+<figure markdown>
+  ![List of APIs][image-8]{ width="100%" }
+</figure>
+
+
+
+The following form will appear. Instructions can be found below on how to find the information required to fill each field on the ServiceNow API form.
+
+Once you have filled out all the fields, press the green 'Save' button, and your API will be connected!
+
+<figure markdown>
+  ![ServiceNow Form][image-1]{ width="100%" }
+</figure>
+
+
+### Name
+Enter any name that will help you recognize this data source from within your query window.
+
+Acceptable characters include:
+
+- lowercase alphanumeric characters
+- underscores
+
+### Instance ID
+
+
+??? example "Steps for finding your API credentials"
+
+    Head over to your ServiceNow [developer portal](https://developer.servicenow.com/dev.do).
```

Comment:
Link this to the sign in page for the developer portal

Reply:
It is